source/_patterns/02-molecules/jumbotron/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Jumbotron
+ */
+
+// Module dependencies
+import base from 'base';
+import image from 'atoms/image';
+import button from 'atoms/button';
+
+// Module styles
+import './_jumbotron.scss';
+
+export default {
+  name: 'jumbotron',
+  deps: [base, image, button],
+  enable($context, { enableHolder }) {
```

Comment:
@jryanconklin remove holder destructuring and logic below. This is only needed in the image component.